include/roboteam_ai/control/controllers/PidTwoAxesController.h

```diff
@@ -0,0 +1,26 @@
+//
+// Created by ratoone on 25-09-19.
+//
+
+#ifndef RTT_PIDTWOAXESCONTROLLER_H
+#define RTT_PIDTWOAXESCONTROLLER_H
+
+#include <include/roboteam_ai/control/ControlUtils.h>
+#include "BaseController.h"
+#include "PidController.h"
+
+class PidTwoAxesController : BaseController<Vector2,Vector2>{
+public:
+    PidController pidXAxis = PidController(0, 0, 0, 0);
```

Comment:
Also not clear for me what these variables mean. But if you document the class then this might be better understandable.